### ISSUE TITLE
Region based music system

### ISSUE THREAD

**Issue by lare96** [luna-rs/luna#91]

**Comment by notjuanortiz** [comment]
This is a large feature that should be broken up.

- [ ] Load music JSON file
- [ ] Cache music data
- [ ] Play/stop music for a single player
- [ ] Lock/unlock music for a single player
- [ ] Support for playing songs based on a player's current region

**Comment by notjuanortiz** [comment]
This is currently blocked by #206.

**Comment by lare96** [comment]
@notjuanortiz Were you planning on taking this on? If not I'll self assign

**Comment by notjuanortiz** [comment]
Eventually but feel free to self assign. There's a ton of other issues/feature that I could work on.

**Comment by lare96** [comment]
Already done locally
